app/models/discount-code.js

```diff
@@ -33,6 +32,9 @@ export default ModelBase.extend({
   isExpired : computed('validTill', function() {
     return new Date() > new Date(this.validTill);
   }),
+  maxQuantity: computed('ticketsNumber', function() {
+    return this.ticketsNumber;
```

Comment:
I don't think we need this, because if the ticketsNumber is changed, this won't change. Please make it a large number for now, like 100000, and open an issue on server to handle the maxQuantity being negative, i.e. -1

If it is negative, server should not limit the discount code count